conjur-oss/README.md

```diff
@@ -104,6 +104,39 @@ $  helm install \
    ./conjur-oss
 ```
 
+### Configuring Conjur Accounts
+
+By setting `account.create` to `true`, you can direct your Conjur
+container to create an admin account during startup. To retrieve the credentials
+for this account, perform the following commands:
+
+```sh-session
+export ACCOUNT_NAME=<Value for `account.name`>
+export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} \
```

Comment:
The syntax `{{ .Release.Namespace }}` works inside of a Helm chart template, but it won't work in a bash shell here. Same for `{{ template "conjur-oss.name" }}' and '{{ .Release.Name }}'. To be consistent with the rest of the README.md, I'd recommend using a dashed phrase in angle brackets, or even better, use bash variables (they don't have to be exported). Also, the `conjur-oss.name` chart name isn't likely to change (it gets its name from the directory that the charts are in) unless someone is doing something advanced with the Helm chart. So the result would be something like this:
```
ACCOUNT_NAME=<conjur-account-name>
CONJUR_NAMESPACE=<conjur-namespace>
HELM_RELEASE=<helm-release>
POD_NAME=$(kubectl get pods --namespace "$CONJUR_NAMESPACE" \
                                    -l "app=conjur-oss,release=$HELM_RELEASE" \
                                    -o jsonpath="{.items[0].metadata.name}")
kubectl exec --namespace "$CONJUR_NAMESPACE" \
              "$POD_NAME" \
              --container=conjur-oss \
              -- conjurctl role retrieve-key "$ACCOUNT_NAME":user:admin
```

Reply:
Yeah this was a typo, my bad!